src/SampleWebApp/Controllers/ControllerWithReadWriteActionsController.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+
+namespace SampleWebApp.Controllers
+{
+    public class ControllerWithReadWriteActionsController : Controller
+    {
+        public IActionResult Index()
+        {
+            return View();
```

Comment:
I don't think it's a good idea because this is really about generating placeholders for you to put your code here. What's the value of a placeholder if you have to reformat it immediately to add code?

Sure some of them might stay trivial, but I personally think it would be annoying.